codeSnippets/snippets/tutorial-full-stack-task-manager/integration-test/src/test/kotlin/IntegrationTest.kt

```diff
@@ -0,0 +1,42 @@
+import core.extension.createConfiguredClient
```

Comment:
Let's add a package to tests. You don't have to put the files into physical packages, just add a correct `package` directive on the top of each file.

Reply:
Could you add a package to all files in the test source set?

Reply:
Yes, I will add the package path to the test file 😄